tools/topology/topology2/include/components/src_format_sxx_to_s32_convert.conf

```diff
@@ -2,345 +2,56 @@
-				# 48khz input
-				{
-					in_rate                 48000
-					in_bit_depth            32
-					in_valid_bit_depth      24
-				}
-				# 64khz input
-				{
-					in_rate                 64000
-					in_bit_depth            32
-					in_valid_bit_depth      24
-				}
-				# 88.2khz input
-				{
-					in_rate                 88200
-					in_bit_depth            32
-					in_valid_bit_depth      24
-				}
-				# 96khz input
-				{
-					in_rate                 96000
-					in_bit_depth            32
-					in_valid_bit_depth      24
-				}
-				# 176.4khz input
-				{
-					in_rate                 176400
-					in_bit_depth            32
-					in_valid_bit_depth      24
-				}
-				# 192khz input
-				{
-					in_rate                 192000
-					in_bit_depth            32
-					in_valid_bit_depth      24
-				}
-				# 8khz input
-				{
-					in_rate			8000
-					in_bit_depth            32
-					in_valid_bit_depth      32
-				}
-				# 11.025 khz input
-				{
-					in_rate                 11025
-					in_bit_depth            32
-					in_valid_bit_depth      32
-				}
-				# 12khz input
-				{
-					in_rate                 12000
-					in_bit_depth            32
-					in_valid_bit_depth      32
-				}
-				# 16khz input
-				{
-					in_rate                 16000
-					in_bit_depth            32
-					in_valid_bit_depth      32
-				}
-				# 22.05khz input
-				{
-					in_rate                 22050
-					in_bit_depth            32
-					in_valid_bit_depth      32
-				}
-				# 24khz input
-				{
-					in_rate                 24000
-					in_bit_depth            32
-					in_valid_bit_depth      32
-				}
-				# 32khz input
-				{
-					in_rate                 32000
-					in_bit_depth            32
-					in_valid_bit_depth      32
-				}
-				# 44.1khz input
-				{
-					in_rate                 44100
-					in_bit_depth            32
-					in_valid_bit_depth      32
-				}
-				# 48khz input
-				{
-					in_rate                 48000
-					in_bit_depth            32
-					in_valid_bit_depth      32
-				}
-				# 64khz input
-				{
-					in_rate                 64000
-					in_bit_depth            32
-					in_valid_bit_depth      32
-				}
-				# 88.2khz input
-				{
-					in_rate                 88200
-					in_bit_depth            32
-					in_valid_bit_depth      32
-				}
-				# 96khz input
-				{
-					in_rate                 96000
-					in_bit_depth            32
-					in_valid_bit_depth      32
-				}
-				# 176.4khz input
-				{
-					in_rate                 176400
-					in_bit_depth            32
-					in_valid_bit_depth      32
-				}
-				# 192khz input
-				{
-					in_rate                 192000
-					in_bit_depth            32
-					in_valid_bit_depth      32
+					in_rate [
+						8000
+						11025
+						12000
+						16000
+						22050
+						24000
+						32000
+						44100
+						48000
+						64000
+						88200
+						96000
+						176400
+						192000
+					]
+					in_bit_depth [
+						16
+						32
+					]
+					in_valid_bit_depth [
+						16
```

Comment:
same thing here. cannot combine the values of 16/32 for bit_depth with 16/24/32 for valid bit depth